data/translations_de.json

```diff
@@ -0,0 +1,412 @@
+    "ProductGroupsTitle": "Liste von product_group_ids",
+    "ProductGroupsDescription": "Gibt eine Liste von product_group_ids an, um dem übergeordneten Element einen Kontext zu geben.",
+    "ProductIdTitle": "Referenz-Token für Produktinstanz",
+    "ProductIdDescription": "Token, das zur Identifizierung eines vollständigen Produktnamens erforderlich ist, damit von anderen Teilen des Dokuments aus auf ihn verwiesen werden kann. Es gibt kein vorgegebenes oder erforderliches Format für die product_id, solange sie ein Produkt im Kontext des aktuellen Dokuments eindeutig identifiziert.",
+    "ProductsTitle": "Liste von product_ids",
+    "ProductsDescription": "Gibt eine Liste von product_ids an, um dem übergeordneten Element einen Kontext zu geben.",
+    "ReferencesItemsTitle": "Referenz",
+    "ReferencesItemsDescription": "Enthält Verweise auf Konferenzen, Papiere, Advisories und andere Ressourcen, die entweder mit einem umgebenden Teil des Dokuments oder mit dem gesamten Dokument in Verbindung stehen und für den Nutzer des Dokuments von Wert sind.",
+    "ReferencesItemsCategoryTitle": "Kategorie der Referenz",
+    "ReferencesItemsCategoryDescription": "Gibt an, ob der Verweis auf das gleiche Dokument oder die gleiche Schwachstelle im Fokus (je nach Geltungsbereich) oder auf eine externe Ressource verweist.",
+    "ReferencesItemsSummaryTitle": "Zusammenfassung der Referenz",
+    "ReferencesItemsSummaryDescription": "Gibt an, worauf sich diese Referenz bezieht.",
+    "ReferencesItemsUrlTitle": "URL der Referenz",
+    "ReferencesItemsUrlDescription": "Gibt die URL der Referenz an.",
+    "VersionTitle": "Version",
+    "VersionDescription": "Gibt eine Versionszeichenfolge an, um die Entstehung des Inhalts des Dokuments eindeutig zu kennzeichnen. Das Format muss entweder Ganzzahl- oder semantische Versionierung sein.",
+    "DocumentTitle": "Metadaten auf Dokumentebene",
+    "DocumentDescription": "Erfasst die Metadaten zu diesem Dokument, das eine bestimmte Menge an Security Advisories beschreibt.",
+    "DocumentAcknowledgmentsTitle": "Danksagungen des Dokuments",
+    "DocumentAcknowledgmentsDescription": "Enthält eine Liste der mit dem gesamten Dokument verbundenen Danksagungen.",
+    "DocumentAggregateSeverityTitle": "Aggregierter Schweregrad",
+    "DocumentAggregateSeverityDescription": "Ist ein Mittel, das vom Hersteller des Dokuments bereitgestellt wird, um die Dringlichkeit und Kritikalität zu vermitteln, mit der eine oder mehrere gemeldete Schwachstellen behoben werden sollten. Es ist eine Metrik auf Dokumentenebene und wird auf das Dokument als Ganzes angewendet - nicht auf eine bestimmte Schwachstelle. Der Wertebereich in diesem Feld wird gemäß den Richtlinien und Verfahren des Dokumenterstellers festgelegt.",
+    "DocumentAggregateSeverityNamespaceTitle": "Namespace des Aggregierten Schweregrads",
```

Comment:
Namespace: I guess we would say "Namensraum" in German?

Reply:
Yes, I changed all occurrences to "Namensraum". I might be a little too comfortable/permissive with Anglicisms, please continue calling me out for that :wink: